Cython/Compiler/FusedNode.py

```diff
@@ -580,6 +580,45 @@ def _unpack_argument(self, pyx_code):
                 {{endif}}
             """)
 
+    def _fused_signature_index(self, pyx_code):
+        """
+        Generate Cython code for constructing a persistent nested dictionary index of
+        fused type specialization signatures.
+        """
+        pyx_code.put_chunk(
+            u"""
+                # CONSTRUCTING NESTED DICTIONARY INDEX OF SPECIALIZED SIGNATURES
+                if not _fused_sigindex:
+                    for sig in <dict>signatures:
+                        sigindex_node = _fused_sigindex
+                        sig_series = sig.strip('()').split('|')
+                        for sig_type in sig_series[:-1]:
+                            if sig_type not in sigindex_node:
+                                sigindex_node[sig_type] = {}
+                            sigindex_node = sigindex_node[sig_type]
+                        sigindex_node[sig_series[-1]] = sig
+            """
+        )
+    def _get_unique_identifier_base(self):
+        """
+        Try to generate a unique identifier string that will likely
+        never be repeated for different nodes, in different processes,
+        on different machines, or in different projects.
+        """
+        # FIXME: Replace and remove this.
+        import time, random, sys
+        return '{time_ns!s}-{id!s}-{randint!s}'.format(time_ns=int(time.time()*1e9), id=id(self), randint=random.randint(0, sys.maxsize-1))
```

Comment:
The code-style test is complaining about long lines.

Reply:
Ah, thank you. I was just thinking about trying to find the code style test.

Reply:
It's surprisingly hard to activate on it's own (or at least worked out how). I think the only way to to de-activate every other type of test...

Reply:
Yeah … there should be a simpler way to get style feedback, I agree …